libcontainer/setns_init_linux.go

```diff
@@ -71,12 +71,13 @@ func (l *linuxSetnsInit) Init() error {
 		unix.Umask(int(*l.config.Config.Umask))
 	}
 
-	if l.config.Config.Scheduler != nil {
-		if err := setupScheduler(l.config.Config); err != nil {
-			return err
-		}
+	if err := setupScheduler(l.config.Config); err != nil {
+		return err
 	}
 
+	if err := setIOPriority(l.config.Config.IOPriority); err != nil {
```

Comment:
Makes sense (although in case of `setupScheduler` we pass the whole config because we also need to access config.Cgroups).

I've also changed its name, as other similar functions have `setup` not `set` prefix.

The change:
```diff
 
-func setIOPriority(ioprio *configs.IOPriority) error {
+func setupIOPriority(config *configs.Config) error {
        const ioprioWhoPgrp = 1
 
+       ioprio := config.IOPriority
        if ioprio == nil {
                return nil
        }
```